README.md

```diff
@@ -1 +1,128 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h1><b>LEADERBOARD</b></h1>
+
+</div>
+
+## 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 LEADERBOARD <a name="about-project"></a>
+
+This LEADERBOARD displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+_[LEADERBOARD]_ is a Javascript, HTML, and CSS project
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Javascript
+- Linters
+- Webpack
+
+### Key Features <a name="key-features"></a>
+
+- [Medium-fidelity Wireframes]
+- [Desktop Version]
+- [Dynamic Design]
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming soon
+
```

Comment:
- [ ] It would be nice if you remove the whole live demo section since you don't have any working link for this app 🤔